tests/unit/Hydrators/FamilyTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Choredo\Test\Hydrators;
+
+use Assert\LazyAssertionException;
+use Choredo\Entities;
+use Choredo\Hydrators;
+use PHPUnit\Framework\TestCase;
+use const Choredo\SHORT_DATA_FIELD_MAX_SIZE;
+
+class FamilyTest extends TestCase
+{
+    protected $dataStub = [
+        'name'            => 'test family',
+        'paymentStrategy' => Entities\Family::PAYMENT_STRATEGY_PER_CHILD,
+        'weekStartDay'    => 'sunday',
+    ];
+
+    public function testHydratorGeneratesIdWhenNotProvided()
+    {
+        $data = $this->dataStub;
+
+        $hydrator = new Hydrators\Family();
+        $family = $hydrator->hydrate($data);
+
+        $this->assertInstanceOf(Entities\Family::class, $family);
+        $this->assertNotEmpty($family->getId());
```

Comment:
Should you also be asserting that it's a valid UUID?

Reply:
:+1: